provision/ubuntu/install.sh

```diff
@@ -162,3 +162,7 @@ chmod +x "bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh"
 sudo -E "./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh"
 sudo -E mv /usr/local/bin/bazel /usr/bin
 rm "bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh"
+
+
+# Kernel parameters
+echo 0 > /proc/sys/kernel/randomize_va_space
```

Comment:
Not an expert here, but did some searching on the topic and according to [1] this will not survive a reboot, and something like this should be done instead:

`echo "kernel.randomize_va_space = 0" > /etc/sysctl.d/01-disable-aslr.conf`

I have not tested this, but this should take effect on every reboot of the image.

[1] https://askubuntu.com/questions/318315/how-can-i-temporarily-disable-aslr-address-space-layout-randomization

Reply:
Furthermore `cat /proc/sys/kernel/randomize_va_space` could be used to verify the value when the CI runs?